context.go

```diff
@@ -40,6 +40,10 @@ type Helper interface {
 	Jid() string
 	JobType() string
 
+	// JobCustom provides access to the job custom hash.
+	// No type checking is performed, please use with caution.
+	JobCustom(name string) interface{}
```

Comment:
How about naming this method `Custom(key string)`?

Reply:
Fine by me. Thanks for the feedback.